pkg/reconciler/pipelinerun/resources/pipelinerunresolution.go

```diff
@@ -580,6 +580,52 @@ func GetTaskRunName(taskRunsStatus map[string]*v1beta1.PipelineRunTaskRunStatus,
 	return kmeta.ChildName(prName, fmt.Sprintf("-%s", ptName))
 }
 
+// GetTaskRunsNames should return unique names for `TaskRuns` if one has not already been defined, and the existing one otherwise.
+func GetTaskRunsNames(taskRunsStatus map[string]*v1beta1.PipelineRunTaskRunStatus, childRefs []v1beta1.ChildStatusReference, ptName, prName string, combinationCount int) []string {
```

Comment:
NIT: the function name sounds a little weird - `GetTaskRunsNames` v/s `GetTaskRunNames` - get the list of the names of `taskruns`. How about `GetNamesofTaskRuns`?